cdisc-org/cdisc-rules-engine
# Update report template and test reporting issues

Steps to test:

1. Run the unit tests
2. Run a validation
3. Verify the following is correct
Issue Summary

    RuleId has been updated to CORE-ID
    Severity has been deleted

Issue Details

    RuleId has been updated to CORE-ID

Conformance Details

    Bundle details header has been changed to Standards Details
    CORE Engine Version has been added
